### DISCUSSION TITLE
Move hitchwiki code to https://github.com/Hitchwiki/hitchwiki

### DISCUSSION THREAD

**Discussion by tillwenke** [Hitchwiki/hitchwiki#198] [General]
@Hitchwiki/admins

**Comment by tillwenke** [comment]
... not leaving credentials in this public repo should be doable :)

**Comment by guaka** [comment]
doable but not trivial, the setup (incl LocalSettings.php) is kinda messy

**Reply by tillwenke** [reply]
it being messy is just another reason to move it here :)

what about dumping the code in a private @Hitchwiki/admins repo so that multiple people can look at it - there might be someone with time & skill. 

to avoid breaches https://github.com/wikimedia/mediawiki/blob/master/.gitignore should be a good start.

**Reply by tillwenke** [reply]
... same would go for trash and nomadwiki actually
